config/locales/es.yml

```diff
@@ -44,6 +44,7 @@ es:
         docs: URL de la documentación
         mail: URL de la lista de correo
         wiki: URL de la Wiki
+        funding: URL de los fondos
```

Comment:
Please don't add the translation of these. just the keys for every locale other than english.